src/os/xdg.zig

```diff
@@ -30,6 +30,22 @@ pub fn config(alloc: Allocator, opts: Options) ![]u8 {
 
 /// Get the XDG cache directory. The returned value is allocated.
 pub fn cache(alloc: Allocator, opts: Options) ![]u8 {
+    // On macOS we should use ~/Library/Caches instead of ~/.cache
```

Comment:
Wouldn't you want to check if XDG_CACHE_HOME is set first?

Reply:
Probably not, because it is difficult to set environment variables for UI applications on MacOS. But if config file lookup uses XDG vars first and then falls back to platform directory, cache should work the same then to keep things consistent...
Using XDG vars on MacOS does not make much sense IMO if proper platform directories are used. They are mostly helpful to redirect non-native apps to proper places.

Reply:
Following current macOS platform conventions (see [FileManager docs](https://developer.apple.com/documentation/foundation/filemanager)), we use` ~/Library/Caches` for cache storage while still respecting `XDG_CACHE_HOME` if set.